src/session.rs

```diff
@@ -131,6 +135,43 @@ pub fn create_batch(queries: Vec<&PreparedStatementWrapper>) -> BatchWrapper {
     BatchWrapper { inner: batch }
 }
 
+fn apply_options(
+    mut query: PreparedStatement,
+    options: &QueryOptionsWrapper,
+) -> napi::Result<PreparedStatement> {
```

Comment:
:wrench: naming nit again: please use `prepared` as the name for `PreparedStatement` variables; `query` should be reserved for unprepared statements.